tests/integration-tests/clusters_factory.py

```diff
@@ -28,6 +30,17 @@
 )
 
 
+def suppress_and_log_exception(func):
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except Exception as e:
+            logging.error(e)
```

Comment:
let's improve this log message to something like: `"Failed when running function {func.__name__}. Ignoring failure. Error: {str(e)}"`